src/Storefront/Resources/views/storefront/component/buy-widget/buy-widget-form.html.twig

```diff
@@ -17,7 +17,6 @@
                     {% block buy_widget_buy_quantity_container %}
                         {% if showQuantitySelect %}
                             <div class="col-4 d-flex justify-content-end">
-                                {% set selectQuantityThreshold = 100 %}
```

Comment:
i guess removing the vars would be a breaking change, as extending blocks can rely on that value being set :thinking: 

IMHO it is best to mark the variables as @deprecated for 6.6.0 and then remove them in that version